VAS.Core/Store/ActionLink.cs

```
@@ -145,7 +145,7 @@ public override int GetHashCode ()
 
 		public override string ToString ()
 		{
-			return string.Format ("{0} -> {1}", SourceButton.Name, DestinationButton.Name);
+			return string.Format ("{0} -> {1}", SourceButton?.Name, DestinationButton?.Name);
```

Why would `SourceButton` and `DestinationButton` be null? Are we hiding a real error here?